package.json

```diff
@@ -25,6 +25,8 @@
     "domain-middleware": "~0.1.0",
     "eslint": "^6.8.0",
     "eslint-config-habitrpg": "^6.2.0",
+    "eslint-config-standard": "^14.1.1",
```

Comment:
I think those were because of the datepicker (needed to link it) - will check and remove if not needed anymore

Reply:
thanks! apart from this the code looks good to me